src/components/Location/LinkedBedListing.tsx

```
@@ -0,0 +1,71 @@
+        const isSelected = selectedLinkedBed?.id === linkedBed.id;
+
+        return (
+          <div
+            key={linkedBed.id}
+            className={cn(
+              "h-32 relative border rounded-lg pt-3 pb-1 cursor-pointer",
+              isSelected && "border-green-600 bg-green-50",
+              !isSelected && "border-gray-400 hover:border-green-200",
+            )}
+            onClick={() => onLinkedBedSelect(linkedBed)}
+          >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)